.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2.1
 executors:
   java:
     docker:
-      - image: azul/zulu-openjdk:8
```

Comment:
it would be better to continue using the zulu jdk for builds, to match what we will use when running this as close as possible

Reply:
that image runs as root by default, we will need to create a new user and login, this resembles running a test locally